gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -1437,6 +1441,57 @@ public default GraphTraversal<S, String> concat(final String... concatStrings) {
         return this.asAdmin().addStep(new ConcatStep<>(this.asAdmin(), concatStrings));
     }
 
+    /**
+     * Returns the value of incoming traverser as strings. Null values are returned as a string value "null".
+     *
+     * @return the traversal with an appended {@link AsStringStep}.
+     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#concat-step" target="_blank">Reference Documentation - Concat Step</a>
```

Comment:
(nitpick) I think these links should not point to the reference docs of the _concat_ step, right?